src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -535,7 +535,11 @@ public function testParseNestedTimestampListAsDateTimeObject($yaml, $year, $mont
         $expected = new \DateTime($yaml);
         $expected->setTimeZone(new \DateTimeZone('UTC'));
         $expected->setDate($year, $month, $day);
-        @$expected->setTime($hour, $minute, $second, 1000000 * ($second - (int) $second));
```

Comment:
This patch is just adding consistency with another test in the same file. Looks like we missed it, maybe during a merge.